doc/user_guide/user_guide.md

```diff
@@ -134,7 +134,9 @@ Rules:
 
 Notice:
 
-* Please, specify the platforms list **without spaces**. This is a valid input:  `github,maven`. This is invalid input: `github, maven`
+* There are two ways to specify multiple platforms: 
+    1. `-p github,maven` 
```

Comment:
I suggest to remove this option as is error prone, and leave only the standardized version

Reply:
I suggest to leave it be, because it's also an officially supported way to provide arguments, and you don't need to use it if you don't want to. At least I don't want to break the interface right now and raise the major version because of this tiny change. It can be removed in future

Reply:
I'll add the deprecation warning after we merge another branch to avoid conflicts in the changes file